dask/bytes/tests/test_compression.py

```diff
@@ -0,0 +1,39 @@
+from io import BytesIO
+
+import pytest
+from dask.bytes.compression import compress, decompress, files
+
+def test_compression():
+    assert set(compress) == set(decompress)
```

Comment:
It doesn't.  It tends to be though.  I don't know a compression library that provides one but not the other.  It's a handy test to have around to avoid mistakes.  Lets remove it if a compression library ever comes up where this won't hold.